test/integration_test.js

```diff
@@ -9,22 +9,69 @@ const properties = {
   'scalar.ledger.client.server_host': 'localhost',
```

Comment:
Sorry, the config namespace and some of the names are updated.
Please refer to this.
https://github.com/scalar-labs/scalar-samples/blob/master/conf/client.properties

I'm wondering why the integration tests pass.

Reply:
Because the constructor of ClientService parses the properties.

Reply:
Ah, right.